core/src/main/scala/chisel3/Data.scala

```diff
@@ -372,6 +372,11 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
     }
   }
 
+  // If this Data is constant, it must hold a constant value
+  private var _isConst:         Boolean = false
+  private[chisel3] def isConst: Boolean = _isConst
+  private[chisel3] def isConst_=(isConst: Boolean) = _isConst = isConst
```

Comment:
Is `isConst_=` used anywhere?

Reply:
Yes, it's used to mark a `Data` as `const` in the `Const.apply()` method.